src/SossMLJ.jl

```diff
@@ -97,12 +97,18 @@ function MMI.predict(sm::SossMLJModel, fitresult, Xnew)
     post = fitresult.post
     pred = predictive(m, keys(post[1])...)
 
-    # predictor = SossPredictor(sm, post, pred, Xnew)
-
     map(Tables.rowtable(Xnew)) do xrow
         args = merge(sm.transform([xrow]), sm.hyperparams)
         SossMLJPredictor(sm, post, pred, args)
     end
 end
 
+function MMI.predict_joint(sm::SossMLJModel, fitresult, Xnew)
```

Comment:
```suggestion
# function MMI.predict_joint(sm::SossMLJModel, fitresult, Xnew)
function predict_joint(sm::SossMLJModel, fitresult, Xnew)
```

`MMI.predict_joint` doesn't actually exist yet upstream. So let's leave this comment as a reminder to ourself.

Reply:
@cscherrer